include/aspect/heating_model/interface.h

```diff
@@ -79,6 +80,19 @@ namespace aspect
         update ();
 
         /**
+         * Function to compute the heating terms in @p heating_outputs given the
+         * inputs in @p material_model_inputs and the outputs of the material
+         * model in @p material_model_outputs.
+         * The default implementation calls specific_heating_rate to make
+         * this implementation backwards compatible.
```

Comment:
add a comment that says that heating_outputs is the value at each quadrature point as defined in inputs.